exercises/D-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return number > 0 && number < 10;
+}
```

Comment:
Hi Kara,

You did great work, congratulation,

I think "between" should include the numbers of 0 and 10 as well, so your code will be:

return number >= 0 && number <= 10;

What do you think?

Reply:
Thank you Ahmad! I have corrected my code :)